### ISSUE TITLE
UnicodeDecodeError at installation

### ISSUE THREAD

**Issue by eight04** [click-contrib/click-spinner#28]
```
Collecting click-spinner
  Using cached https://files.pythonhosted.org/packages/4d/c4/e4cc6595266021b549f
774a20329188e68b63f06896eba94c1c952027c6e/click-spinner-0.1.8.tar.gz
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\owner\appdata\local\programs\python\python37\python.exe'
 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Owner\\AppD
ata\\Local\\Temp\\pip-install-2phu8zfb\\click-spinner\\setup.py'"'"'; __file__='
"'"'C:\\Users\\Owner\\AppData\\Local\\Temp\\pip-install-2phu8zfb\\click-spinner\
\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read()
.replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"
'"'exec'"'"'))' egg_info --egg-base 'C:\Users\Owner\AppData\Local\Temp\pip-insta
ll-2phu8zfb\click-spinner\pip-egg-info'
         cwd: C:\Users\Owner\AppData\Local\Temp\pip-install-2phu8zfb\click-spinn
er\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Owner\AppData\Local\Temp\pip-install-2phu8zfb\click-spinner
\setup.py", line 6, in <module>
        readme = f.read()
    UnicodeDecodeError: 'cp950' codec can't decode byte 0xe2 in position 373: il
legal multibyte sequence
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check th
e logs for full command output.
```

I think it is fixed by #23 but is never released? This bug prevents Windows from installing this package.

**Comment by jayvdb** [comment]
Yes, the fix hasnt been released
https://github.com/click-contrib/click-spinner/compare/v0.1.8...master

ping @yoavram

**Comment by jayvdb** [comment]
@click-contrib/maintainers Could one of you help out here to get a release happening?

**Comment by jayvdb** [comment]
Hmm. that didnt link, I guess I dont have rights to do it.
https://github.com/orgs/click-contrib/teams/maintainers also doesnt appear to have the same effect.
ping @timofurrer

**Comment by timofurrer** [comment]
Unfortunately, I don't have any permissions on this repository. 
I'm checking with the maintainers Team, if any one can help here! Also, PyPI permissions are an entirely different story. 

I'll keep you posted here :tada:

**Comment by yoavram** [comment]
Sorry, haven't been checking this for a while. How can I help??

**Comment by jayvdb** [comment]
Basically Windows users experience will be better if there is a new release of already merged fixes.

**Comment by yoavram** [comment]
I released v0.1.9
